sdkproject/Assets/Mapbox/Unity/Utilities/Conversions.cs

```diff
@@ -200,13 +200,13 @@ public static RectD TileBounds(UnwrappedTileId unwrappedTileId)
 		/// <param name="longitude"> The longitude. </param>
 		/// <param name="zoom"> Zoom level. </param>
 		/// <returns> A <see cref="T:UnityEngine.Vector2d"/> xy tile ID. </returns>
```

Comment:
Please don't forget to edit the code comments too.
Are there no other changes needed?
That would mean `LatitudeLongitudeToTileId` is not referenced anywhere in our code -> this is bad.
We definitely need tests for `Conversions.cs`.

Reply:
@BergWerkGIS Sorry for being an idiot, will edit code comments now. The return type was Vector2d and in most cases we would only use the x,y component of the tileId. We found this issue when writing an API for queryingHeight at a certain LatLong for which we needed the entire tileid.